.github/workflows/windows.yml

```
@@ -11,12 +11,26 @@ jobs:
         ruby: [ mingw, mswin, 3.0, 2.7, 2.6, 2.5, 2.4 ]
     steps:
       - uses: actions/checkout@v2
+      - name: Install libraries with vcpkg
+        run: |
+          vcpkg --triplet x64-windows install libyaml
+      - name: link libraries
+        run: |
+          for %%I in (C:\vcpkg\installed\x64-windows\bin\*.dll) do (
+            mklink %%~nxI %%I
+          )
```

Need to install libyaml with vcpkg only on mswin, I think.
```suggestion
      - name: Install libraries with vcpkg
        run: |
          vcpkg --triplet x64-windows install libyaml
        if: ${{ matrix.ruby != 'mswin' }}
      - name: link libraries
        run: |
          for %%I in (C:\vcpkg\installed\x64-windows\bin\*.dll) do (
            mklink %%~nxI %%I
          )
        if: ${{ matrix.ruby != 'mswin' }}
```